### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by SreyPichh**

### CODE REVIEW

File: counter/Classes/Presentation/Scenes/Main/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+    
+    override func viewDidLayoutSubviews() {
+        self.subview.updatedConstraints()
+    }
+}
+extension ViewController: StoryboardView {
+    func bind(reactor: ViewControllerReactor) {
+        self.subview.addBtn.rx.tap
+            .map{Reactor.Action.addBtnClicked}
```
Comment (kenphanith) [inline counter/Classes/Presentation/Scenes/Main/ViewController.swift:38]: pay attention to the indentation

File: counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        case minusBtnClicked
+    }
+    enum Mutation {
+        case add
+        case minus
+    }
+    struct State {
+        var counter: Int
+    }
+    var initialState: ViewControllerReactor.State
```
Comment (kenphanith) [inline counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift]: normally we initialize State in this line 
`
var initialState: ViewControllerReactor.State = State(
     counter: 0
)
`

File: counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    init() {
+        initialState = State(counter: 0)
```
Comment (kenphanith) [inline counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift]: init maybe mainly use from constructor injection with private members

**Review by kenphanith — CHANGES_REQUESTED** [review]

File: counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift (hunk 2 above)
Comment (SreyPichh) [reply counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift]: yeah yesterday you told me, I will correct it

**Review by SreyPichh — COMMENTED** [review]
changed

**Comment by kenphanith** [comment]
LGTM

**Comment by kenphanith** [comment]
P.S: When we agree to merge the pull request u can merge and delete the branch

**Pull request merged**